skops/hub_utils/tests/test_hf_hub.py

```diff
@@ -271,6 +271,23 @@ def test_init(classifier_pickle, config_json):
         )
 
 
+def test_init_modelcard_creation(classifier_pickle, config_json):
+    # create a temp directory and delete it, we just need a unique name.
+    dir_path = tempfile.mkdtemp()
+    shutil.rmtree(dir_path)
```

Comment:
Why do you make and remove a directory path?

Reply:
because we need a directory name / path, but we don't want it to exist before init.